README.md

```diff
@@ -1 +1,49 @@
-# WIP
\ No newline at end of file
+## prometheus-cachet-glue
+
+A simple lightweight daemon that takes in webhook calls from prometheus' alertmanager and
+sets the state of a component according to that.
+
+##### Limitations
+Currently this expects the alerts array to be exactly one element.
```

Comment:
What happens if more than one comes in? Does it do the first one but not the remainder? What could cause the alerts array to have more than one element?

Reply:
@jcgruenhage What did we do about this?

Reply:
I think we concluded that we should be able to prevent this on the alertmanager side, but I forgot to confirm that that is the case.

Reply:
From how I read https://prometheus.io/docs/alerting/alertmanager/#grouping we should be able to prevent this.

If you want I could also refactor the code a bit and iterate over `alert.alerts` though instead of just getting `alert.alerts[0]`

Reply:
Perhaps just a test on array length and raise an error or something? If alertmanager is configurable the 'right way' then doing a trust-but-verify step in this code base will stop typo's

Reply:
So if there is more than one entry just 500 the request with a body explaining why?

Reply:
Done :heavy_check_mark: